frontend/src/styles/GlobalStyle.ts

```diff
@@ -4,8 +4,8 @@ export const globalStyle = kialiStyle({
   height: '100%',
   margin: 0,
   padding: 0,
-  // TODO: possible change to --pf-global--FontFamily--redhat-updated--sans-serif
-  fontFamily: 'var(--pf-global--FontFamily--sans-serif)',
+  // TODO: possible change to --pf-v5-global--FontFamily--redhat-updated--sans-serif
+  fontFamily: 'var(--pf-v5-global--FontFamily--sans-serif)',
```

Comment:
I think this might be the culprit of the weird fonts. Is this var available in pf5? Maybe it needs to be set to something else or it has been split into a few different vars? I can't seem to find pf documentation on this.

Reply:
yep...they make some changes I am going to try it new options

 --pf-v5-global--FontFamily--text: "RedHatText", helvetica, arial, sans-serif;
  --pf-v5-global--FontFamily--heading: "RedHatDisplay", helvetica, arial, sans-serif;
  --pf-v5-global--FontFamily--monospace: "RedHatMono", "Liberation Mono", consolas, "SFMono-Regular", menlo, monaco, "Courier New", monospace;
  --pf-v5-global--FontFamily--text--vf: "RedHatTextVF", "RedHatText", helvetica, arial, sans-serif;
  --pf-v5-global--FontFamily--heading--vf: "RedHatDisplayVF", "RedHatDisplay", helvetica, arial, sans-serif;
  --pf-v5-global--FontFamily--monospace--vf: "RedHatMonoVF", "RedHatMono", "Liberation Mono", consolas, "SFMono-Regular", menlo, monaco, "Courier New", monospace;
  --pf-v5-global--FontSize--4xl: 2.25rem;
  --pf-v5-global--FontSize--3xl: 1.75rem;
  --pf-v5-global--FontSize--2xl: 1.5rem;
  --pf-v5-global--FontSize--xl: 1.25rem;
  --pf-v5-global--FontSize--lg: 1.125rem;
  --pf-v5-global--FontSize--md: 1rem;
  --pf-v5-global--FontSize--sm: 0.875rem;
  --pf-v5-global--FontSize--xs: 0.75rem;
  --pf-v5-global--FontWeight--normal: 400;
  --pf-v5-global--FontWeight--bold: 700;

Reply:
The moved from **"RedHatText","Overpass",overpass,helvetica,arial,sans-serif;** to "RedHatText", helvetica, arial, sans-serif. They removed overpass